vsgenstubs4/_vapoursynth.part.pyi

```diff
@@ -73,49 +73,316 @@
+        'TRANSFER_IEC_61966_2_1', 'TRANSFER_BT2020_10', 'TRANSFER_BT2020_12', 'TRANSFER_ST2084', 'TRANSFER_ARIB_B67',
+
+    'ColorPrimaries', 'PRIMARIES_BT709', 'PRIMARIES_UNSPECIFIED',
+        'PRIMARIES_BT470_M', 'PRIMARIES_BT470_BG', 'PRIMARIES_ST170_M', 'PRIMARIES_ST240_M', 'PRIMARIES_FILM',
+        'PRIMARIES_BT2020', 'PRIMARIES_ST428', 'PRIMARIES_ST431_2', 'PRIMARIES_ST432_1', 'PRIMARIES_EBU3213_E',
+
+    # Environment SubSystem
+    'Environment', 'EnvironmentData',
+
+    'EnvironmentPolicy',
+        'StandaloneEnvironmentPolicy',
+        'VSScriptEnvironmentPolicy',
+
+    'EnvironmentPolicyAPI',
+    'register_policy', 'has_policy',
+
+    'get_current_environment',
+
+    'VideoOutputTuple',
+    'clear_output', 'clear_outputs', 'get_outputs', 'get_output',
+
+    '_construct_type', '_construct_parameter', 'construct_signature',
+
+    # Logging
+    'PythonVSScriptLoggingBridge', 'LogHandle', 'Error',
+
+    # Functions
+    'FuncData', 'Func', 'CallbackData', 'FramePtr',
+    'Plugin', 'Function',
+
+    # Formats
+    'VideoFormat',
+
+    # Frames
+    'RawFrame', 'VideoFrame', 'AudioFrame',
+    'FrameProps',
+
+    # Nodes
+    'RawNode', 'VideoNode', 'AudioNode',
+
+    'Core', '_CoreProxy', 'core',
+
+    # Inspection API [UNSTABLE API]
+    # '_try_enable_introspection'
+]
+
+
+###
+# Typing
+
+T = TypeVar('T')
+S = TypeVar('S')
+
+SingleAndSequence = Union[T, Sequence[T]]
+
+
+@runtime_checkable
+class SupportsString(Protocol):
+    @abstractmethod
+    def __str__(self) -> str:
+        ...
+
+
+DataType = Union[str, bytes, bytearray, SupportsString]
+
+_VapourSynthMapValue = Union[
+    SingleAndSequence[int],
+    SingleAndSequence[float],
+    SingleAndSequence[DataType],
+    SingleAndSequence['VideoNode'],
+    SingleAndSequence['VideoFrame'],
+    SingleAndSequence['AudioNode'],
+    SingleAndSequence['AudioFrame'],
+    SingleAndSequence['VSMapValueCallback[Any]']
+]
+
+BoundVSMapValue = TypeVar('BoundVSMapValue', bound=_VapourSynthMapValue)
+
+VSMapValueCallback = Callable[..., BoundVSMapValue]
+
+
+class _Future(Generic[T]):
+    def set_result(self, value: T) -> None: ...
+
+    def set_exception(self, exception: BaseException) -> None: ...
+
+    def result(self) -> T: ...
+
+    def exception(self) -> Union[NoReturn, None]: ...
+
+###
+# Typed dicts
+
+
+class _VideoFormatInfo(TypedDict):
+    id: int
+    name: str
+    color_family: 'ColorFamily'
+    sample_type: 'SampleType'
+    bits_per_sample: int
+    bytes_per_sample: int
+    subsampling_w: int
+    subsampling_h: int
+    num_planes: int
+
+
+###
+# VapourSynth Versioning
+
+
+class VapourSynthVersion(NamedTuple):
+    release_major: int
+    release_minor: int
+
+
+class VapourSynthAPIVersion(NamedTuple):
+    api_major: int
+    api_minor: int
+
+
+__version__: VapourSynthVersion
+__api_version__: VapourSynthAPIVersion
+
+###
+# VapourSynth Cython capsule
+
+
+class PyCapsule(Structure):
+    ...
+
+
+__pyx_capi__: Dict[str, PyCapsule]
```

Comment:
Thgis is internal cython API. I don't want this published within the .pyi file. If at all, please provide a vapoursynth.pxd file for this.